drools-wb-screens/drools-wb-scenario-simulation-editor/drools-wb-scenario-simulation-editor-backend/src/main/java/org/drools/workbench/screens/scenariosimulation/backend/server/DMNTypeServiceImpl.java

```diff
@@ -82,6 +83,14 @@ public FactModelTuple retrieveFactModelTuple(Path path, String dmnPath) {
         return factModelTuple;
     }
 
+    @Override
+    public Simulation initializeNameAndNamespace(Simulation simulation, Path path, String dmnPath) {
```

Comment:
@danielezonca 
I would expect an "initialize" method to return a _void_; moreover, since it returns the same instance given as parameter, returning it is confusing. 
A lot of similar method (i.e. methods that accept an instance of a given type and returns the same type) actually returns new/different instances; that's the reason of above comment.

I think there are two options to makes it clearer:
1) return Simulation without accepting one (i.e. create a new one inside the method) - if possible;
2) return void - so it is clear that the method change the status of the provided Simulation.

Reply:
Done as option 2